src/base/static/components/story-sidebar/index.js

```diff
@@ -0,0 +1,190 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Map, OrderedMap, fromJS } from "immutable";
+
+import StoryChapter from "./story-chapter";
+import constants from "../../constants";
+
+import "./index.scss";
+
+import { story as storyConfig, place as placeConfig } from "config";
+
+class StorySidebar extends Component {
+  constructor(props) {
+    super(props);
+
+    this.stories = Object.keys(storyConfig).reduce((memo, storyName) => {
+      return memo.set(
+        storyName,
+        Object.keys(storyConfig[storyName].order).reduce((memo, chapterUrl) => {
```

Comment:
Closing over the same variable name is confusing. Can we name these `memo`'s to something more specific?